h2o-r/h2o-package/R/models.R

```diff
@@ -4079,8 +4078,7 @@ h2o.genericModel <- function(model_file_path){
 #' }
 #' @export
 h2o.import_mojo <- function(mojo_file_path){
-  model_file_key <- h2o.importFile(mojo_file_path, parse = FALSE)
-  model <- h2o.generic(model_key = model_file_key)
+  model <- h2o.generic(path = mojo_file_path)
   print(model)
```

Comment:
should this print be here?

Reply:
No there is no need, same case below, let's remove it.